app/javascript/mastodon/components/account_bio.tsx

```diff
@@ -44,13 +46,13 @@ export const AccountBio: React.FC<AccountBioProps> = ({
   }
 
   return (
-    <div
+    <AnimateEmojiProvider
       className={`${className} translate`}
       onClickCapture={handleClick}
       ref={handleNodeChange}
     >
```

Comment:
`AnimateEmojiProvider` doesn't forward refs yet, so I think this ref won't be attached properly

Reply:
fixed, but ironically my next step would be to replace that `ref` as it's trying to manipulate the DOM of React elements, which is a no-no.